api/spec/requests/spree/api/orders_controller_spec.rb

```diff
@@ -317,6 +317,21 @@ module Spree
           expect(credit_cards[0]['id']).to eq payment.source.id
           expect(credit_cards[0]['address']['id']).to eq credit_card.address_id
         end
+
+        it 'renders the payment source view for gateway' do
+          subject
+          expect(response).to render_template partial: "spree/api/payments/source_views/_#{payment.payment_method.partial_name}"
```

Comment:
These are sufficient, but should be more explicit. Either `render_template('spree/api/payments/source_views/_gateway.html.erb`), or test directly for the right thing being rendered as JSON.